lib/internal/Magento/Framework/Stdlib/DateTime/Timezone/LocalizedDateToUtcConverter.php

```diff
@@ -65,9 +62,7 @@ public function convertLocalizedDateToUtc($date)
         $localTimestamp = $formatter->parse($date);
         $gmtTimestamp = $this->timezone->date($localTimestamp)->getTimestamp();
         $formattedUniversalTime = date($this->defaultFormat, $gmtTimestamp);
-
-        $date = new \DateTime($formattedUniversalTime, new \DateTimeZone($configTimezone));
-        $date->setTimezone(new \DateTimeZone('UTC'));
+        $date = new \DateTime($formattedUniversalTime, new \DateTimeZone('UTC'));
```

Comment:
UTC timezone is redundant since Magento setup UTC by default
https://github.com/magento-tsg/magento2ce/blob/2.4-develop/app/bootstrap.php#L73

Reply:
Fixed.